client/jetpack-cloud/sections/jetpack-social/promo.jsx

```diff
@@ -32,7 +32,7 @@ export const Promo = ( { isSocialActive, adminUrl, translate } ) => {
 
 	return (
 		<Main wideLayout className="jetpack-social__promo">
-			<DocumentHead title={ titleHeader } />
+			<DocumentHead title={ titleHeader } />{ ' ' }
```

Comment:
It looks like a bunch of spaces got added to this file, I'm assuming by mistake. Maybe an auto-linter or something added these?

Reply:
I'd say yes remove them, I don't see a need for them to be there 🤔

Reply:
I think Prettier added those. I removed those characters and uninstalled Prettier :D Please let me know if you see anything else.

Reply:
Got it 👍  Do you think this is ready to be deployed once the Social team says?